libs/component-library/src/lib/tabs/tabs.tsx

```diff
@@ -1,60 +1,113 @@
-import { useTheme } from '@shopify/restyle';
 import React from 'react';
 import { FxBox, FxReanimatedBox } from '../box/box';
-import { FxPressableOpacity } from '../pressable-opacity/pressableOpacity';
+import {
+  FxPressableOpacity,
+  FxPressableOpacityProps,
+} from '../pressable-opacity/pressableOpacity';
 import { FxText, FxTextProps } from '../text/text';
-import { FxTheme } from '../theme/theme';
 import {
   useAnimatedReaction,
   useAnimatedStyle,
-  useDerivedValue,
   useSharedValue,
   withTiming,
 } from 'react-native-reanimated';
+import { FxSpacer } from '../spacer/spacer';
+import { StyleSheet, View } from 'react-native';
+import { ColorProps } from '@shopify/restyle';
+import { FxTheme } from '../theme/theme';
+
+const ANIMATION_DURATION = 150;
+
+type ColorOverrideType = ColorProps<FxTheme> & {
+  color: keyof FxTheme['colors'];
+};
+
+const SELECTED_TEXT: ColorOverrideType = { color: 'greenBase' };
+const AUTO_PRESSED: ColorOverrideType = { color: 'border' };
+const FIXED_PRESSED: ColorOverrideType = {
```

Comment:
Should these be in the theme file so it is easy to update colours all at once?

Reply:
I'll create a separate ticket to revisit this.